metricbeat/helper/kubernetes/ktest/ktest.go

```diff
@@ -39,22 +60,54 @@ func GetTestCases(files []string) ptest.TestCases {
 				// Example:
 				//	Metricsfile: ../_meta/test/ksm.v2.4.2
```

Comment:
Is this  example right? I think the metric file is in ../_meta/test/KSM/ksm.v.2.4.2.plain